Amplify/Categories/Auth/Request/AuthChangePasswordRequest.swift

```diff
@@ -28,6 +28,9 @@ public extension AuthChangePasswordRequest {
 
     struct Options {
 
+        // TODO: Move this metadata to plugin options. All other request has the metadata
```

Comment:
Please add an issue for this. I know we as a team haven't reached consensus regarding putting TODOs inline in code, but a GH issue would be helpful in tracking.

Reply:
Add GH issue - https://github.com/aws-amplify/amplify-ios/issues/902